src/content/1.7/development/webservice/tutorials/3-prestashop-webservice-lib/1-install-library.md

```diff
@@ -0,0 +1,117 @@
+composer install
+```
+
+Or if you already have an existing project simply add the dependency:
+
+```bash
+composer require "Prestashop/Prestashop-webservice-lib"  
+```
+
+The library is not PSR compliant and has no namespace therefore you need to update your `composer.json` file to include the class in your `autoload`:
+
+```json
+{
+    "require": {
+        "Prestashop/Prestashop-webservice-lib": "dev-master"
+    },
+    "autoload": {
+        "files": [
+            "vendor/prestaShop/prestaShop-webservice-lib/PSWebServiceLibrary.php"
+        ]
+    }
+}
+```
+
+And regenerate your `autoload`:
+
+```bash
+composer dump-autoload
+```
+
+Then you can use the library via autoloading:
+
+```php
+<?php
+
+require_once('./vendor/autoload.php');
+
+$webService = new PrestaShopWebservice(...);
+```
+
+### Manual install
+
+If you don't use composer you can download the [library archive](https://github.com/PrestaShop/PrestaShop-webservice-lib/archive/master.zip) and extract it where you need.
+You will then need to manually load the `PSWebServiceLibrary` in your PHP script or application:
+
+```php
+require_once('./PSWebServiceLibrary.php');
+
+$webService = new PrestaShopWebservice(...);
+```
+
+## Accessing the webservice
+
+### Create your client
+
+First, you must create an instance of the `PrestaShopWebservice` object, which takes 3 parameters in its constructor:
+
+- The store's root path (ex: http://example.com/ ).
+- The authentication key (ex: ZR92FNY5UFRERNI3O9Z5QDHWKTP3YIIT).
+- A boolean value, indicating whether the Web service must use its debug mode.
+
+```php
+$webService = new PrestaShopWebservice('http://example.com/', 'ZR92FNY5UFRERNI3O9Z5QDHWKTP3YIIT', false);
+```
+
+Once the instance is created you can access the following methods:
+
+| Method   | HTTP equivalent   | SQL    |
+|----------|-------------------|--------|
+| get()    | POST              | INSERT |
+| add()    | GET               | SELECT |
+| edit()   | PUT               | UPDATE |
+| delete() | DELETE            | DELETE |
+
+### Handling errors
+
+It is essential that you understand how to handle errors with the web service library. By implementing error-catch method early, you will more easily detect issues, and be able to correct them on the go.
+
+Error handling with the web service library is done using PHP exceptions. If you do not know about them, you should read http://php.net/manual/en/language.exceptions.php, as exceptions are an essential part of good coding practice.
+
+The error handling is done within a `try..catch` block, with the web service instanciation and execution being done in the `try` section, the `catch` one containing the error handling code. There are many types of exception that exist you can catch specific ones which allows you to deal with each error case accordingly, the webservice library uses `PrestaShopWebserviceException` so you can catch this one only to deal with errors related to webservice.
```

Comment:
Not necessary to explain the try catch block, you already linked the php doc

Reply:
Well it doesn't hurt to emphasize it 😅 Since not everyone will click on the link, it's a 30s read..